<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHtmlClass</class>
 <widget class="QWidget" name="QHtmlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QHtml</string>
  </property>
  <widget class="QPushButton" name="startbutton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>0</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>承兑人</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>金额(万元)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到期日</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>每十万扣息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>瑕疵</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>交易平均时长</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>日志</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="LogOutput">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="filterList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>银行</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>金额（最小）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>金额（最大）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>年息</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>每十万扣息</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余时间(最小)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余时间(最大)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addFilterParamBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加筛选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setFilterParamBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置筛选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setFilterParamBtn_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除筛选</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QHtml.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>setFilterParamBtn_2</sender>
   <signal>clicked()</signal>
   <receiver>QHtmlClass</receiver>
   <slot>DelteFilterParam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>924</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>907</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DelteFilterParam()</slot>
 </slots>
</ui>
